lib/plugins.js

```diff
@@ -190,6 +190,11 @@ function plugins (settings, deviceId) {
       .then(row => row.id)
   }
 
+  function fetchCurrentAvailableCoupons () {
+    const sql = `SELECT * FROM coupons WHERE soft_deleted=false`
```

Comment:
check with Rafael if it's not better to count directly in the DB by using COUNT() instead